ThunderBasics/UIColor+Helpers.swift

```diff
@@ -10,6 +10,31 @@ import UIKit
 
 extension UIColor {
     
+    /// A structual representation of a colour's RGBA components
+    public struct RGBAComponents: Equatable {
+        
+        public let red: CGFloat
```

Comment:
Does it matter if the components, say `red`, are in `[0, 255]` or `[0, 1]`.
If so, shall we add this information to the docs?